src/segments/ui5tooling_test.go

```diff
@@ -0,0 +1,168 @@
+package segments
+
+import (
+	"fmt"
+	"oh-my-posh/environment"
+	"oh-my-posh/mock"
+	"oh-my-posh/properties"
+	"path/filepath"
+	"testing"
+
+	"github.com/alecthomas/assert"
```

Comment:
qhy the additional assert? We use `github.com/stretchr/testify/assert` by default.

Reply:
I missed this, I'll correct it.